tokio-console/src/view/controls.rs

```diff
@@ -129,13 +129,21 @@ impl ControlDisplay {
 }
 
 /// Returns a list of controls which are available in all views.
-pub(crate) fn universal_controls() -> &'static Vec<ControlDisplay> {
-    static UNIVERSAL_CONTROLS: OnceCell<Vec<ControlDisplay>> = OnceCell::new();
-
-    UNIVERSAL_CONTROLS.get_or_init(|| {
-        vec![
-            ControlDisplay::new_simple("toggle pause", "space"),
-            ControlDisplay::new_simple("quit", "q"),
-        ]
-    })
+const fn universal_controls() -> &'static [ControlDisplay] {
```

Comment:
nit, not a big deal: can't this just be 
`const UNIVERSAL_CONTROLS: &'static [ControlDisplay]`?

Reply:
Yes it can! Changed.